src/SonarScanner.MSBuild/Program.cs

```diff
@@ -44,7 +44,7 @@ private static async Task<int> Execute(string[] args)
         public static async Task<int> Execute(string[] args, ILogger logger)
         {
             Utilities.LogAssemblyVersion(logger, Resources.AssemblyDescription);
-#if NET46
+#if NET462
             logger.LogInfo("Using the .NET Framework version of the Scanner for MSBuild");
```

Comment:
Consider using [RuntimeInformation.FrameworkDescription](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.frameworkdescription?view=net-7.0) instead/additionally. It is .Net471 and above, so it may not help much anyway.

Reply:
Good point. We should consider this if we move to 4.8